<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="PddRules">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>621</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1. Общие положения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2. Общие обязанности водителей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3. Применение спец. сигналов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4. Обязанности пешеходов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5. Обязанности пассажиров</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6. Сигналы светофора и регулировщика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7. Применение аварийной сигнализации и знака аварийной остановки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8. Начало движения, маневрирование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9. Расположение транспортных средств на проезжей части</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10. Скорость движения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11. Обгон, опережение, встречный разъезд</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12. Остановка и стоянка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13. Проезд перекрестков</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14. Пешеходные переходы и места остановок маршрутных транспортных средств</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15. Движение через железнодорожные пути</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16. Движение по автомагистралям</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17. Движение в жилых зонах</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18. Приоритет маршрутных транспортных средств</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19. Пользование внешними световыми приборами и звуковыми сигналами</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20. Буксировка механических транспортных средств</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21. Учебная езда</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22. Перевозка людей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23. Перевозка грузов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24. Дополнительные требования к движению велосипедистов и водителей мопедов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25. Дополнительные требования к движению гужевых повозок, а также к прогону животных </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26. Нормы времени управления транспортным средством и отдыха</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lab1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Правила дорожного движения </string>
    </property>
   </widget>
   <widget class="QPushButton" name="go2rules">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Перейти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go2signs">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Перейти</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дорожные знаки и разметка</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PddSigns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>621</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1. Предупреждающие знаки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2. Знаки приоритета</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3. Запрещающие знаки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4. Предписывающие знаки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5. Знаки особых предписаний</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6. Информационные знаки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7. Знаки сервиса</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8. Знаки дополнительной информации (таблички)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1. Горизонтальная разметка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2. Вертикальная разметка</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="PddDopusk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>621</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Основные положения по допуску транспортных средств к эксплуатации и обязанности должностных лиц по обеспечению безопасности дорожного движения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Перечень неисправностей и условий, при которых запрещается эксплуатация транспортных средств</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="go2dopusks">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Перейти</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab1_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Допуск и неисправности т.с</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab1_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Билеты ПДД 2021</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab1_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Актуально на 01.11.2021</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab1_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Экзамен категории ABM</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PddVariants">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Вариант 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 33</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 34</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 36</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 37</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 38</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 39</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вариант 40</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="go2exams">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Приступить к решению</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go2bilets">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Приступить к решению</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
